pull_request_template.md

```diff
@@ -1,52 +1,54 @@
 ## Purpose
-> Describe the problems, issues, or needs driving this feature/fix and include links to related issues in the following format: Resolves issue1, issue2, etc.
+> Creatio has REST API layer to process various kind of objects of the Creatio with Cookies based authentication. When the integrtaion implement with WSO2 EI, it needs to implement Authentication from the scratch, therefore readymade solution will overcome this issue. 

```

Comment:
Please refrain from changing this template. This is the template that needs to be filled when sending a PR.

Reply:
Reverted.